eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,25 @@
+parameters: 
+  TargetUserVariable: 'notspecified'
```

Comment:
Should these be notspecified or just be empty?

Reply:
empty, otherwise we will override with variable name with `notspecified.`